crates/task-impls/src/consensus.rs

```diff
@@ -1253,17 +1217,17 @@ where
                     Ok(None) => {
                         debug!("We were not chosen for consensus committee on {:?}", view);
                     }
-                    Ok(Some(vote_token)) => {
-                        let message = self
-                            .timeout_exchange
-                            .create_timeout_message::<I>(view, vote_token);
-
-                        debug!("Sending timeout vote for view {}", *view);
-                        if let GeneralConsensusMessage::TimeoutVote(vote) = message {
-                            self.event_stream
-                                .publish(HotShotEvent::TimeoutVoteSend(vote))
-                                .await;
-                        }
+                    Ok(Some(_vote_token)) => {
```

Comment:
Do we plan to refactor `make_vote_token` so that it won't return an unused `vote_token` anymore? No need to make the change in this PR though.

Reply:
Yeah that's exactly the plan.  I'm working on that at the moment, will be the next PR